### PR TITLE
Mockup UIs

### PR DESCRIPTION
**Pull request by nghianguyen-terra**
@tailehuu  Please review

### CODE REVIEW

**Comment by nghianguyen-terra** [comment]
@tailehuu  Please review

**Comment by nghianguyen-terra** [comment]
@tailehuu  Please review

File: css/style.scss [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+            }
+        }
+
+        .medkumo-sdk-clear {
+            clear: both;
+        }
+    }
+}
+
+.medkumo-sdk-book-an-appointment .form-row > label span {
```
Comment (tailehuu) [inline css/style.scss]: this one should under _.medkumo-sdk-body_ ???

File: css/style.scss [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+            position: absolute;
+            left: 10px;
+        }
+    }
+
+    .medkumo-sdk-body {
+        width: 100%;
+        padding: 10px;
+
+        .user-detail {
```
Comment (tailehuu) [inline css/style.scss]: .medkumo-sdk-user-detail ???

File: css/style.scss [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+        .medkumo-sdk-book-an-appointment {
+            max-width: 640px;
+            margin: 0 auto;
+            padding: 55px;
+            box-sizing: border-box;
+            background-color: #ffffff;
+            font: bold 14px sans-serif;
+            text-align: center;
+
+            h1 {
```
Comment (tailehuu) [inline css/style.scss]: user-detail-h1 ??

**Comment by nghianguyen-terra** [comment]
@tailehuu Please review

File: css/style.scss (hunk 3 above)
Comment (nghianguyen-terra) [reply css/style.scss]: Done

**Review by nghianguyen-terra — COMMENTED** [review]
@tailehuu Please review

**Comment by nghianguyen-terra** [comment]
@tailehuu Please review help me

File: css/style.scss [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+            float: none;
+            margin-top: 0;
+            text-align: center;
+        }
+
+        #medkumo-sdk-book-an-appointment-form-row-btnBack {
+          margin: 25px 67px;
+        }
+
+        .medkumo-sdk-book-an-appointment input {
```
Comment (tailehuu) [inline css/style.scss]: .medkumo-sdk-book-an-appointment {
   input {
      width: 190px;
   }
}

File: css/style.scss [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+
+    .medkumo-sdk-header {
+        position: relative;
+        height: 30px;
+        line-height: 30px;
+        background: #6caee0;
+        color: white;
+        min-width: 100px;
+
+        .medkumo-sdk-header-close-button {
```
Comment (tailehuu) [inline css/style.scss]: .medkumo-sdk-close-button

File: css/style.scss [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+            position: absolute;
+            right: 10px;
+        }
+
+        .medkumo-sdk-header-maximize-button,
```
Comment (tailehuu) [inline css/style.scss]: .medkumo-sdk-maximize-button

File: css/style.scss [hunk 7]
```diff
@@ -0,0 +1,200 @@
...
+        .medkumo-sdk-header-minimize-button {
```
Comment (tailehuu) [inline css/style.scss]: .medkumo-sdk-minimize-button

File: css/style.scss [hunk 8]
```diff
@@ -0,0 +1,200 @@
...
+        }
+
+        .medkumo-sdk-list-of-doctor {
+            float: left;
+            text-align: center;
+            max-width: 527px;
+            padding: 0 10px;
+            box-sizing: border-box;
+
+            .item{
```
Comment (tailehuu) [inline css/style.scss]: .medkumo-sdk-item

File: css/style.scss [hunk 9]
```diff
@@ -0,0 +1,200 @@
...
+              float: left;
+              margin: 5px;
+
+              .btn_Style {
```
Comment (tailehuu) [inline css/style.scss]: .medkumo-sdk-book-an-appointment-button

File: css/style.scss [hunk 10]
```diff
@@ -0,0 +1,200 @@
...
+                background-color: #6caee0;
+                color: #ffffff;
+                font-weight: bold;
+                box-shadow: 1px 2px 4px 0 rgba(0, 0, 0, 0.08);
+                padding: 14px 22px;
+                border: 0;
+                /*margin: 0 183px 0;*/
+            }
+
+            .medkumo-sdk-book-an-appointment-form-row{
```
Comment (tailehuu) [inline css/style.scss]: .medkumo-sdk-form-row

File: css/style.scss [hunk 11]
```diff
@@ -0,0 +1,200 @@
...
+                text-align: left;
+                margin-bottom: 20px;
+
+                #medkumo-sdk-book-an-appointment-form-row-btnBook {
```
Comment (tailehuu) [inline css/style.scss]: #medkumo-sdk-form-row-book-button

File: css/style.scss [hunk 12]
```diff
@@ -0,0 +1,200 @@
...
+                    margin-left: 183px;
+                    margin-right: 25px;
+                }
+
+                #medkumo-sdk-book-an-appointment-form-row-btnBack {
```
Comment (tailehuu) [inline css/style.scss]: #medkumo-sdk-form-row-back-button

File: css/style.scss [hunk 13]
```diff
@@ -0,0 +1,200 @@
...
+                }
+            }
+
+        }
+
+        .medkumo-sdk-clear {
+            clear: both;
+        }
+
+        .medkumo-sdk-book-an-appointment-form-row> label span {
```
Comment (tailehuu) [inline css/style.scss]: why put it here ?

File: css/style.scss [hunk 14]
```diff
@@ -0,0 +1,200 @@
...
+
+
+/*	Making the form responsive. Remove this media query
+    if you don't need the form to work on mobile devices. */
+@media (max-width: 600px) {
+    .medkumo-sdk-book-an-appointment {
+        padding: 30px;
+        max-width: 480px;
+
+        .medkumo-sdk-book-an-appointment-form-row {
```
Comment (tailehuu) [inline css/style.scss]: .medkumo-sdk-form-row

File: css/style.scss [hunk 15]
```diff
@@ -0,0 +1,200 @@
...
+            max-width: 300px;
+            margin: 25px auto;
+            text-align: left;
+
+            #medkumo-sdk-book-an-appointment-form-row-btnBack {
```
Comment (tailehuu) [inline css/style.scss]: #medkumo-sdk-form-row-back-button

File: css/style.scss [hunk 16]
```diff
@@ -0,0 +1,200 @@
...
+                margin-left: 0;
+                float: right;
+                margin-right: 0;
+            }
+
+            #medkumo-sdk-book-an-appointment-form-row-btnBook {
```
Comment (tailehuu) [inline css/style.scss]: #medkumo-sdk-form-row-book-button

File: css/style.scss [hunk 17]
```diff
@@ -0,0 +1,200 @@
...
+                margin-left: 0;
+            }
+
+            button {
+                margin: 0;
+            }
+
+            .medkumo-sdk-book-an-appointment .form-row > label span {
```
Comment (tailehuu) [inline css/style.scss]: why put it here?

File: css/style.scss [hunk 18]
```diff
@@ -0,0 +1,200 @@
...
+        width: 100%;
+        overflow-y: scroll;
+
+        .medkumo-sdk-user-detail {
+            float: none;
+            margin-top: 0;
+            text-align: center;
+        }
+
+        #medkumo-sdk-book-an-appointment-form-row-btnBack {
```
Comment (tailehuu) [inline css/style.scss]: #medkumo-sdk-form-row-back-button

**Comment by nghianguyen-terra** [comment]
@tailehuu Please review

**Pull request merged**